Common/YamlParsers/Models/InstallSection.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections.Generic;
+using System.Linq;
+using JetBrains.Annotations;
+
+namespace Common.YamlParsers.Models
+{
+    public class InstallSection
+    {
+        public InstallSection(object install, object artifacts, object artefacts)
+        {
+            Install = Transform(install);
+            Artifacts = Transform(artifacts);
```

Comment:
Same field with 2 names. Merge it during the parsing.

Reply:
1. It's obviously a typo. There should be only one variant in the end.
2. on a configuration level it's still two separate sections. All the merging happens in Mergers. Introducing a merging logic to a high-level parser is not appropriate and redundant considering the previous point.

Reply:
No, the InstallSection model should obviously contains only one of them.

Reply:
It's not obvious.
It's easier to deprecate `artefacts` and only support the `artifacts` field, as there is no mention of `artefacts` in the [docs](https://github.com/skbkontur/cement/blob/master/README-module.yaml.md#moduleyaml).

The logic for merging `artifacts` and `artefacts` now lives in a single place - `InstallSectionParser`.
You suggesting to copypaste this logic to top-level parsers: `ModuleDefaultsParser` and `ConfigSectionParser`. For what?

Reply:
Ok, let's not support the wrong `artefacts` in new parsers.

Reply:
the last one :)

Reply:
I guess the deprecation process should take time.

First, we can proactively search for module.yamls with `artefacts` (that's actually two modules out of 897).
Then, we replace old parsers with the new ones. The new ones would write a warning about `artefact` field.
And only afterwards it's safe to delete `artefacts` support.

Reply:
I've submitted two merge-requests for those two modules.
https://git.skbkontur.ru/ke/forms.clientinfo/merge_requests/2
https://git.skbkontur.ru/ke/formats/merge_requests/5

Reply:
Yes, just fix them.
Also there is no critical effect of this section.

Reply:
Alright, I've removed 'artefacts` field support.